test/apps/iodemo/io_demo.cc

```diff
@@ -462,6 +470,9 @@ class P2pDemoCommon : public UcxContext {
                 break;
 #endif
             case UCS_MEMORY_TYPE_HOST:
+                if (_memh && _context) {
+                    assert(_context->unmap_buffer(_memh) == UCS_OK);
```

Comment:
please don't call function which must be called in the parameter of assert(), this is not safe. For example if assert is defined as blank macro, then the calling of the function would be missed.